src/server/game/Entities/Object/Object.h

```diff
@@ -627,6 +628,10 @@ class TC_GAME_API WorldObject : public Object, public WorldLocation
         // transports
         Transport* m_transport;
 
+        // zone/area ID updated in Unit::Update (only Units can move)
```

Comment:
Hm, should we do the call in their `::Update` too? Or are transports always their own area?